src/UAParser.jl

```diff
@@ -276,75 +358,80 @@ function parseos(user_agent_string::AbstractString)
 return OSResult("Other", nothing, nothing, nothing, nothing) #Fail-safe if no match
 end #End parseos
 
-#Vectorize parseos for any array of user-agent strings
-Base.@vectorize_1arg AbstractString parseos
 
 ##############################################################################
 ##
 ## Extend DataFrames to include UAParser methods
 ##
 ##############################################################################
 
-#Convenience function to take nothing type to UTF8String of length 0
+#Convenience function to take nothing type to String of length 0
 #This is a hack for sure
-function nothing_to_utf8empty(x::Union{AbstractString, Void})
+function nothing_to_emptystr(x::Union{AbstractString, Void})
   if x == nothing
     x = ""
   else
-    x = convert(UTF8String, x)
+    x = convert(String, x)
   end
 end
 
 #DeviceResult to DataFrame method
 function DataFrame(x::Array{DeviceResult, 1})
-  temp = DataFrame([element.family for element in x])
-  names!(temp, ["device"])
-  return temp
+    temp = DataFrame(String, size(x, 1), 3)
+    names!(temp, ["device", "brand", "model"])
+
+    temp["device"] = String[ξ.family for ξ ∈ x]
+
+    temp["brand"] = String[nothing_to_emptystr(ξ.brand) for ξ ∈ x]
```

Comment:
This use of unicode seems excessive, given the simplicity of the list comprehension. My preference is to put this back to ASCII, so that the next developer who comes along and may be less mathematically inclined will not have to understand the set syntax you are using.